site/content/docs/install_aio.md

```diff
@@ -0,0 +1,451 @@
+---
+title: "SNAS AIO Install"
+date: 2017-01-04T15:04:10.000Z
+---
+
+SNAS All-In-One (AIO) Docker Install Steps
+===========================================
+
+<!--more-->
+
+AIO container includes all components except SNAS UI. 
+
+See the various requirements and suggested system configurations at [Requirements](/docs/install).
+
+
+
+Install Using Docker
+--------------------
+
+### docker hub: openbmp/aio
+All-in-one (aio) includes everything needed to run the collector and store the data in MySQL.   You can use this container to test/evaluate SNAS as well as run smaller deployments.  Production deployments normally would have distributed collectors and a redundant pair of MySQL/MariaDB servers. 
+
+#### Container Includes
+* **Openbmpd** - Latest collector (listening port is TCP 5000)
+* **MariaDB 10.0** - MySQL server (listening port TCP 3306)
+* **Apache Kafka 0.8.x** - High performing message bus (listening ports are TCP 2181 and 9092)
+* **Tomcat/DB_REST** - Latest Rest interface into MySQL/MariaDB (listening port TCP 8001)
+* **Openbmp MySQL Consumer** - Latest Consumer that puts all data into MySQL
+
+### Recommended Current Linux Distributions
+
+  1. Ubuntu 14.04/Trusty
+  1. CentOS 7/RHEL 7
+
+- - -
+### 1) Install docker
+Docker host should be **Linux x86_64**.   Follow the [Docker Instructions](https://docs.docker.com/installation/) to install docker.  
+
+- - -
+
+### 2) Download the docker image
+
+    docker pull openbmp/aio
+
+- - -
+### 3) Create MySQL volumes
+MySQL/MariaDB uses a shared container (host) volume so that if you upgrade, restart, change the container it doesn't loose the DB contents.  **The DB will be initialized if the volume is empty.**  If the volume is not empty, the DB will be left unchanged.  This can be an issue when the schemas need to change. Therefore, to reinit the DB and apply the latest schema use docker run with the ```-e REINIT_DB=1```
```

Comment:
Let's remove the "REINIT_DB" reference from all text.  This isn't needed and instead is causing confusion. 

Instead we can add a section for "How to reinitialize the DB".